website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -11,22 +11,20 @@ versioned mode. For general information about the usage and operation of the kv
 secrets engine, please see the [Vault kv
 documentation](/vault/docs/secrets/kv).
 
-~> Note: This documentation assumes the kv secrets engine is enabled at the
```

Comment:
I removed this note, as we no longer use `secret` as a placeholder for the parameter